recipe/build.sh

```diff
@@ -1,16 +1,6 @@
 #!/bin/bash
-set -ex
 
 export DISABLE_AUTOBREW=1
 
-# arrow uses C++17
-export ARROW_R_CXXFLAGS="${ARROW_R_CXXFLAGS} -std=c++17"
-export LIBARROW_BUILD=false
-
-if [[ "${target_platform}" == osx-* ]]; then
-    # See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
-    export ARROW_R_CXXFLAGS="${ARROW_R_CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
-fi
-
-# ${R_ARGS} necessary to support cross-compilation
-${R} CMD INSTALL --build r/. ${R_ARGS}
+# shellcheck disable=SC2086
+${R} CMD INSTALL --build . ${R_ARGS}
```

Comment:
I'm very confused what the bot is doing here @beckermr 

How can it ever be a valid migration to blindly(?) overwrite build scripts?

Reply:
This is a bug clearly. Please open an issue.

Reply:
The old R build.sh boilerplate was so convoluted ([example](https://github.com/conda-forge/r-rstanarm-feedstock/blob/fe14a87b1ef1e8ee23295d3580e1a18626e01dcb/recipe/build.sh)), I expect overwriting was the easier path forward.